chainer/functions/loss/softmax_cross_entropy.py

```diff
@@ -100,7 +100,7 @@ def _is_chainerx_supported(self, input_arrays):
             return False
         if self.ignore_label != -1:
             return False
-        if self.reduce != 'mean':
+        if self.reduce == 'mean' and self.normalize:
             return False
```

Comment:
These are the default configurations. How about instead of falling back, run the previous implementation in this case? The denominator https://github.com/chainer/chainer/pull/8250/files#diff-7f9fda8b243258115ef7bdac145fec8dL122 should be `mask.sum()` instead of `x.shape[0]` though.

Reply:
There was a bug with `normalize` which I fixed in https://github.com/chainer/chainer/pull/8301. Maybe we should wait for that one to be merged before.

Reply:
Sounds good. I'll review #8301.